upstream.go

```diff
@@ -603,6 +622,8 @@ func (uc *upstreamConn) handleMessage(msg *irc.Message) error {
 				delete(ch.Members, msg.Prefix.Name)
 				ch.Members[newNick] = membership
 			}
+
+			uc.Logfn(ch.Name, "*** %s is now known as %s", msg.Prefix.Name, newNick)
```

Comment:
We should only log this if the user is part of the channel members.

Reply:
Right